BPA/Ch.12/12.2.2_Covariates.R

```diff
@@ -31,7 +31,7 @@ inits <- lapply(1:nc, function(i)
 
 ## Call Stan from R
 out <- stan("binmix_cov.stan",
-            data = list(y = y, R = R, T = T, K = 100),
+            data = list(y = y, X=X, R = R, T = T, K = 100),
```

Comment:
For consistency of coding style,  `X = X` (spaces around operators) would be preferable.

Reply:
Good point. Fixed now.